src/sage/libs/giac/giac.pyx

```diff
@@ -880,14 +880,14 @@ cdef class Pygen(GiacMethods_base):
         sig_on()
         t=GIAC_taille(self.gptr[0], 6000)
         sig_off()
-        if (t<6000) :
+        if t < 6000:
             sig_on()
-            result=decstring23(GIAC_print(self.gptr[0], context_ptr).c_str()) #python3
+            result = decstring23(GIAC_print(self.gptr[0], context_ptr).c_str()) #python3
```

Comment:
the comment `#python3` could be removed. At least an extra space before is needed

Reply:
You removed the next occurence of `#python3` but not this one...